examples/resource/README.md

```diff
@@ -1,13 +1,62 @@
 # Resource detection Example
 
-An example application that shows what resource attributes will be detected.
+An example application that shows what resource attributes will be detected by the [GCP Resource Detector](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/gcp-resources) and how the [Autoconfigure Resource Provider SPI](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#resource-provider-spi) automatically *attaches* the detected resource attributes to the generated telemetry.
```

Comment:
It would be nice to explicitly state at the top that this does not generate any telemetry, as most users trying out an example probably expect to find traces, metrics, or logs.